reconcile/gitlab_housekeeping.py

```diff
@@ -59,6 +64,40 @@
 DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 
 
+merged_merge_requests = Counter(
+    name="qontract_reconcile_merged_merge_requests",
+    documentation="Number of merge requests that have been successfully merged in a repository",
+    labelnames=["project_id"],
+)
+
+rebased_merge_requests = Counter(
+    name="qontract_reconcile_rebased_merge_requests",
+    documentation="Number of merge requests that have been successfully rebased in a repository",
+    labelnames=["project_id"],
+)
+
+time_to_merge = Histogram(
+    name="qontract_reconcile_time_to_merge_merge_request",
```

Comment:
Would you consider adding the unit  as a metric suffix?

Reply:
Good catch! I've added it.